README.md

```diff
@@ -102,8 +103,9 @@ distributions.
 
 ```bash
 opx-build/scripts/opx_run opx_rel_pkgasm.py --dist stable \
-  -b opx-onie-installer/release_bp/OPX_dell_base.xml
+  -b opx-onie-installer/release_bp/OPX_dell_base_<os_release>.xml
 ```
+**Note:** In the above command change **os_release** to jessie or stretch
```

Comment:
Don't use OS Release. Since we are dealing with two different releases (Debian and OPX), use `distribution` for Debian releases and `release` for OPX releases.